response_operations_ui/controllers/collection_instrument_controllers.py

```diff
@@ -308,6 +308,31 @@ def _create_ci_version_map(registry_instruments):
     return {ci["classifier_value"]: ci["ci_version"] for ci in registry_instruments}
 
 
+def delete_registry_instruments(collection_exercise_id: str, form_type: str) -> str:
+    url = (
+        f'{app.config["COLLECTION_INSTRUMENT_URL"]}/collection-instrument-api/1.0.2/'
+        f"registry-instrument/exercise-id/{collection_exercise_id}/formtype/{form_type}"
+    )
+
+    response = requests.delete(url, auth=app.config["BASIC_AUTH"])
+
+    try:
+        response.raise_for_status()
+        log_message = "Successfully deleted collection instrument from registry instruments"
```

Comment:
Not sure about referring to the collection instrument here, its the registry instrument we are deleting or the collection instrument version. May make this confusing in the future.

Reply:
As above.